clang/lib/Format/UnwrappedLineFormatter.cpp

```diff
@@ -32,6 +32,24 @@ bool isRecordLBrace(const FormatToken &Tok) {
                      TT_StructLBrace, TT_UnionLBrace);
 }
 
+bool LineStartsNamespaceScope(const AnnotatedLine* Line,
+                              const AnnotatedLine* PreviousLine,
+                              const AnnotatedLine* PrevPrevLine) {
+  return PreviousLine && PreviousLine->Last->is(tok::l_brace) &&
```

Comment:
The existence of `PrevPrevLine` suggest that `PreviousLine` has to exist. Just assert on that.